.github/ISSUE_TEMPLATE/hcl-report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Report Hardware Compatibility List
+about: 'Report information about compatible hardware that you are using in your setup'
+title: ''
+labels: 'P: default, T: hcl'
+assignees: ''
+
+---
+
+**Dasharo version**
+<!--(The version of Dasharo you're using (e.g., `v0.2.0`))-->
+
+**CPU**
+
+* Processor name: <!--(full name including vendor, brand and CPU number))-->
+* Core name: <!--(CPU core codename))-->
+* CPU base speed: <!--(based CPU speed))-->
+* CPU boost spped - <!--(boosted CPU speed))-->
+* Wattage: <!--(declared by producer processor wattage))-->
+* GPU: <!--(information about embedded graphics processing unit))-->
+* Results: <!--(link to measurement results))-->
```

Comment:
what is ` measurements results` here?

Reply:
copy paste from dasharo docs - it shouldn't be here